xbmc/guilib/GUIIncludes.cpp

```diff
@@ -530,6 +539,8 @@ const INFO::CSkinVariableString* CGUIIncludes::CreateSkinVariable(const std::str
 {
   std::map<std::string, TiXmlElement>::const_iterator it = m_skinvariables.find(name);
   if (it != m_skinvariables.end())
-    return INFO::CSkinVariable::CreateFromXML(it->second, context);
+    return INFO::CSkinVariable::CreateFromXML(it->second, context, [&](const std::string &expression) -> std::string {
```

Comment:
`ResolveExpressions` is a non-static member function, so...

 For usage in the lambda the current object has to be captured. `[this]` will work, though.

And to pass it directly it needs to be bound to the current object first, and I think
`std::bind(&CGUIIncludes::ResolveExpressions, this, std::placeholders::_1, std::vector<std::string>())`
is less readable than the lambda.